FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
- - opening_brace
+ - todo
+
+opt_in_rules:
+ - anyobject_protocol
+ - array_init
+ - attributes
+ - closure_body_length
+ - closure_end_indentation
+ - closure_spacing
+ - collection_alignment
+ - conditional_returns_on_newline
+ - contains_over_filter_count
+ - contains_over_filter_is_empty
+ - contains_over_first_not_nil
+ - contains_over_range_nil_comparison
+ - convenience_type
+ - discouraged_optional_boolean
+ - discouraged_optional_collection
+ - empty_collection_literal
+ - empty_count
+ - empty_string
+ - empty_xctest_method
+ - enum_case_associated_values_count
+ - expiring_todo
+ - explicit_init
+ - explicit_self
+ - extension_access_modifier
+ - fatal_error_message
+ - file_header
+ - file_name_no_space
+ - first_where
+ - flatmap_over_map_reduce
+ - force_unwrapping
+ - function_default_parameter_at_end
+ - identical_operands
+ - implicit_return
+ - implicitly_unwrapped_optional
+ - indentation_width
+ - joined_default_parameter
+ - last_where
+ - legacy_multiple
+ - legacy_random
+ - let_var_whitespace
+ - literal_expression_end_indentation
+ - lower_acl_than_parent
+ - modifier_order
+ - multiline_arguments
+ - multiline_literal_brackets
+ - multiline_parameters
+ - object_literal
+ - operator_usage_whitespace
+ - optional_enum_case_matching
+ - overridden_super_call
+ - override_in_extension
+ - pattern_matching_keywords
+ - prefer_self_type_over_type_of_self
+ - prefer_zero_over_explicit_init
+ - prohibited_interface_builder
+ - reduce_into
+ - redundant_nil_coalescing
+ - redundant_type_annotation
+ - sorted_first_last
+ - sorted_imports
```

Comment:
Not sure about `sorted_imports`. It is good in Java world, but in Apple world there aren't a lot of imports thanks to visibility in targets.
I'm not aginst this rule, but sometimes it is convenient to have something like this (made up example):
```swift
import UIKit
import BackgroundTasks

// Analytics
import Firebase
import FirebaseCore

// Logging
import os.log
```

Reply:
When there are not many of them, it's fine to not to do sorting. But it starts to be messy when there are more than 5.
The issue is it's hard to catch the moment when you need to enforce sorting. So, I'd like to enforce from the beginning